conf/distro/include/omnect-os-distro.conf

```diff
@@ -1,11 +1,23 @@
+TARGET_VENDOR = "-omnect"
 DISTRO = "omnect-os"
 DISTRO_NAME = "${DISTRO_NAME_PREFIX}OMNECT${DISTRO_NAME_SUFFIX}"
-DISTROOVERRIDES = "poky:omnect-os"
-# DISTRO_VERSION is POKY_VERSION (since we include poky conf before our own
-# distro settings) appended with a build number.
-DISTRO_VERSION:append = ".${OMNECT_BUILD_NUMBER}"
-DISTRO_FEATURES += "polkit"
-TARGET_VENDOR = "-omnect"
+DISTRO_VERSION = "${OE_VERSION}.${OMNECT_BUILD_NUMBER}"
+DISTRO_FEATURES += "debuginfod ipv4 ipv6 polkit seccomp zeroconf"
```

Comment:
we set DISTRO_FEATURES via kas.
i just looked, since we append them there, it's okay to set "=" here, otherwise it's not.
i would like to leave it as is.

Reply:
done